neurobooth_os/tasks/task.py

```diff
@@ -158,6 +160,22 @@ def __init__(
             units="deg",
         )
 
+    def render_image(self):
```

Comment:
Fine with this at the moment, but I think there should be a way to better handle this. (The Task class is taking on a lot of roles/responsibilities.)